### DISCUSSION TITLE
FBA Inventory Event Detail Report snapshot date how represent?

### DISCUSSION THREAD

**Discussion by rohitdobariya** [amzn/selling-partner-api-models#3375] [General]
I an have issue with one of the UK seller,
If I download the GET_FBA_FULFILLMENT_INVENTORY_SUMMARY_DATA for one month in the single report there is no issue with it,
If I download the same report daily I get more record compare to one month single report.
I noticed that daily report downloads contain more than one timezone offset and also these are wired timezone offset
snapshotDate
2022-07-01T01:00:00+01:00
2022-07-01T02:00:00+02:00
2022-07-02T02:00:00+02:00
2022-07-03T02:00:00+02:00
2022-07-06T05:30:00+05:30
2022-07-15T04:00:00+04:00
See how wired timezone offset is time in an hour and minutes are always the same as offset, why I am not sure.

If I download report single report for a month it will contain only one timezone offset but wired timezone offsets are also there.

The marketplace is A1F83G8C2ARO7P

In summary
If I download the report for 01 July to 31 July I get 1500 records with the same timezone offset
If I download the report for July month (Each day generates a report for the previous day ) Total I get 2009 records with different timezone offset
Nots : StartDateTime and EndDateTime I provided in UTC format.

I am trying to find out the issues last few days as US seller reports are working properly.
